developer-docs-site/docs/cli-tools/build-from-brew.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Build CLI from Brew"
+id: "build-aptos-cli-brew"
+---
+
+# Build Aptos CLI from Brew
+
+On macOS and some Linux, `brew` is a package manager that allows for installing and updating packages in a single 
+command.
+
+:::tip Not supported on Windows
+Brew is not supported fully on Windows
+:::
+
+## Installation
+
+1. Ensure you have `brew` installed https://brew.sh/
+2. Open a terminal and enter the following commands
+```bash
+    brew update        # Gets the latest updates for packages
+    brew install aptos # Installs the Aptos CLI
+```
+3. You can now get help instructions by running `aptos help`. You may have to open a new terminal window.
```

Comment:
Just double checking not `aptos --help` right?

Reply:
Eh, both do the same thing, looks like it automerged